### ISSUE TITLE
Move the match-history to the virtualized list to display its entries.

### ISSUE THREAD

**Issue by 2Pacalypse-** [ShieldBattery/ShieldBattery#1071]

**Comment by 2Pacalypse-** [comment]
Not sure if this is still preferable versus infinite scrolling pagination? Doesn't seem like it, imo.

**Comment by tec27** [comment]
Not quite sure what you're asking, if we're displaying a lot of entries (more than a screen's worth) then we probably want it to be a virtualized list

**Comment by 2Pacalypse-** [comment]
Oh I'm saying we have pagination on this page now. We load only 40 games at a time as you scroll down, so not sure if we still need the virtualized list at that point? I guess if you scroll a lot and load a lot of games, it could be useful?

**Comment by tec27** [comment]
yes, anything with infinite scrolling should definitely be virtualized
